src/V3Dead.cpp

```diff
@@ -286,6 +286,7 @@ class DeadVisitor final : public VNVisitor {
     }
 
     //-----
+    void visit(AstClockingItem* nodep) override {}
```

Comment:
I added this to prevent `V3Dead` from deleting clockvars that are seemingly dead before `V3AssertPre`. Later the vars are moved to the containing module so if they are actually dead they still get deleted. Added comment explaining this.